src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/init/AMDamageSources.java

```diff
@@ -0,0 +1,10 @@
+package com.github.minecraftschurlimods.arsmagicalegacy.common.init;
+
+import net.minecraft.world.damagesource.DamageSource;
+
+public interface AMDamageSources {
```

Comment:
Why are these plain DamageSources? (why not functions returning IndirectEntityDamageSource/EntityDamageSource like fireball and such)

Reply:
Changed to EntityDamageSource/IndirectEntityDamageSource.